src/applications/resources-and-support/hooks/useGetSearchResults.js

```diff
@@ -2,10 +2,31 @@
 import { useEffect, useState } from 'react';
 import { orderBy } from 'lodash';
 // Relative imports.
+import environment from 'platform/utilities/environment';
 import recordEvent from 'platform/monitoring/record-event';
-import { SEARCH_IGNORE_LIST } from '../constants';
+import {
+  EXPERIMENTAL_SEARCH_IGNORE_LIST,
+  SEARCH_IGNORE_LIST,
+} from '../constants';
 
-const countInstancesFound = (searchableString = '', keyword = '') =>
+const countKeywordsFound = (searchableString = '', keyword = '') => {
+  let keywordsFound = 0;
+
+  // Change the searchable string into a list of words.
+  const searchableWords = searchableString.toLowerCase()?.split(' ');
+
+  // Iterate over each word and see if we have any keywords that match it.
+  searchableWords?.forEach(word => {
```

Comment:
I wrote this with `.reduce` at first but I thought this was more readable

Reply:
The rest of us thank you greatly